utils/views.py

```diff
@@ -17,4 +17,12 @@ def post_usersave(*args, **kwargs):
     chat_id = kwargs["chat_id"]
     data, count = supabase.table("user_info").insert({"username":user_name, "utyreg": uyt_reg, "chatid": chat_id}).execute()
 
+def post_aadhar_update(*args, **kwargs):
```

Comment:
aadhar no and dob are the result of a single-sequenced action. It's better to use a single function for saving this data. ig there is no advantage in using two separate functions of this.

Reply:
requested changes were made